### PR TITLE
Fix ActionView::Helpers::DateHelper#distance_of_time_in_words test

### PR DESCRIPTION
**Pull request by yui-knk**
### Summary

The behavior of `#round` was changed to round to even in Ruby 2.4.
So we have to pass `half: :up` to `#round`.

see: https://github.com/ruby/ruby/commit/6ed8c79ddb11ccfb580bb0a22b22cc1362250255 and
     https://github.com/ruby/ruby/commit/295f60b94d5ff6551fab7c55e18d1ffa6a4cf7e3

And for back compatibility, adding core_ext `#round` to accept options.

### CODE REVIEW

**Comment by rafaelfranca** [comment]
O_O. Are they aware of this breaking change?

**Comment by rafaelfranca** [comment]
Is this the only place where we use `round`?

File: activesupport/lib/active_support/core_ext/numeric/round.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# We have depended a behavior of rounding to nearest.
+# In some case (i.e. <tt>ActionView::Helpers::DateHelper#distance_of_time_in_words</tt>),
+# we have to pass <tt>half: :up</tt> to <tt>round</tt>.
+#
+# This core_ext enables to us to pass a option,
+# even before Ruby 2.3.
+if !(0.0.round(0, half: :up) rescue false)
+  module ActiveSupport::NumericRoundWithOption
+    def round(ndigits = 0, half: nil)
+      super(ndigits)
```
Comment (matthewd) [inline activesupport/lib/active_support/core_ext/numeric/round.rb]: Accepting the option and then ignoring it seems quite rude.

**Comment by rafaelfranca** [comment]
I opened a bug to get their feedback because changing this in a minor version seems weird https://bugs.ruby-lang.org/issues/12958

**Comment by yui-knk** [comment]
Also https://github.com/rails/rails/pull/27177 . This PR is now useless. So I will close this.

**Pull request closed without merging**